kasa/iot/iotlightstrip.py

```diff
@@ -83,6 +90,8 @@ def effect_list(self) -> list[str] | None:
         Example:
             ['Aurora', 'Bubbling Cauldron', ...]
         """
+        # LightEffectModule returns effect names along with a LIGHT_EFFECTS_OFF value
+        # so return the original effect names here for backwards compatability
```

Comment:
```suggestion
        # so return the original effect names here for backwards compatibility
```

Reply:
Would it make sense to return a dict of (name, effect data) in the common interface?

Reply:
Fixed